app/views/stages/_fields.html.erb

```diff
@@ -61,6 +61,17 @@
     </div>
   </div>
 
+  <div class="form-group">
+    <div class="col-sm-offset-2 col-sm-10">
+      <div class="checkbox">
+        <%= form.label :duplicable do %>
+          <%= form.check_box :duplicable %>
+          This stage can serve as a production template.
```

Comment:
then say it "Duplication template" ... or "Clone template" ... or name the
column clone_template ... just somehow have the description and column in
sync ...

On Fri, Aug 12, 2016 at 4:01 PM, Adam Panzer <notifications@github.com>
wrote:

> In app/views/stages/_fields.html.erb
> <https://github.com/zendesk/samson/pull/1198#discussion_r74672133>:
>
> > @@ -61,6 +61,17 @@
> >      </div>
> >    </div>
> >
> > +  <div class="form-group">
> > +    <div class="col-sm-offset-2 col-sm-10">
> > +      <div class="checkbox">
> > +        <%= form.label :duplicable do %>
> > +          <%= form.check_box :duplicable %>
> > +          This stage can serve as a production template.
>
> So here's the deal:
>
>    1.
>
>    At one point in time, I cloned a stage and ran with it, not realizing
>    it was somebody just messing around.
>    2.
>
>    That stage had bundle -v foo bar blah and so it just spit out the
>    current version of bundler. Having a "gold master" stage would fix this
>    problem. If you can come up with a better name, I'm all ears, this was the
>    best thing @sandlerr <https://github.com/sandlerr> and I came up with.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/zendesk/samson/pull/1198/files/122035a3f8635be3fd046f6620704db40db5d7c0#r74672133>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAAsZx9HsmowCIA2_Fq2blVDvxyePgbKks5qfPtGgaJpZM4Jhtog>
> .
>